### PR TITLE
New version: Oracle.VirtualBox version 7.0.4 [FP-O]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Oracle VM VirtualBox | Oracle VM VirtualBox 7.0.4    |
| Version     | 7.0.4     | 7.0.4 |
| Publisher   | Oracle Corporation   | Oracle and/or its affiliates      |
| ProductCode |           | {AC025744-F936-446B-A128-AC42F974FCC8}    |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [3967](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3497263617>)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
cc @stephengillie

**Review by ItzLevvie — DISMISSED** [review]

**Comment by stephengillie** [comment]
@vedantmgoyal2009 

This package has a dependency on one or more Microsoft C++ Redistributable packages. When running in a clean sandbox, it failed to install due to the absence of the Microsoft Visual C++ 2019 Redistributable package. Can you add the correct [Microsoft.VCRedist](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/VCRedist) package as a package dependency to the manifest? Here is an [example](https://github.com/microsoft/winget-pkgs/blob/92b8d5d7727aa641aa3d758dbd653a4c8ba4ec8c/manifests/v/VMware/WorkstationPro/16.2.3/VMware.WorkstationPro.installer.yaml#L20-L23).

**Review by dorssel — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by stephengillie** [comment]
image
Dependencies may need to be adjusted.

File: manifests/o/Oracle/VirtualBox/7.0.2/Oracle.VirtualBox.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+PackageName: Oracle VM VirtualBox
+PackageUrl: https://www.virtualbox.org
+License: GPL-3.0-only
+LicenseUrl: https://www.virtualbox.org/wiki/Licensing_FAQ
+Copyright: Copyright © 1995, 2022, Oracle and/or its affiliates.
+CopyrightUrl: https://www.oracle.com/legal/terms.html
+ShortDescription: VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use.
+Description: |-
+  VirtualBox is a powerful x86 and AMD64/Intel64 virtualization product for enterprise as well as home use.
+  Not only is VirtualBox an extremely feature rich, high performance product for enterprise customers, it is also the only professional solution that is freely available as Open Source Software under the terms of the GNU General Public License (GPL) version 2.
```
Comment (dorssel) [inline manifests/o/Oracle/VirtualBox/7.0.2/Oracle.VirtualBox.locale.en-US.yaml:21]: License was fixed above (GPL-3.0-only). But the description here still refers to GPL version 2.

**Review by dorssel — CHANGES_REQUESTED** [review]

**Comment by Sn1cket** [comment]
It's now over a month since this PR was created. Is there something one can help with to get this finally merged.

**Comment by stephengillie** [comment]
The installer has a dependency on VCRedist. 
image

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/o/Oracle/VirtualBox/7.0.4/Oracle.VirtualBox.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+PackageName: Oracle VM VirtualBox
+PackageUrl: https://www.virtualbox.org
+License: GPL-3.0-only
+LicenseUrl: https://www.virtualbox.org/wiki/Licensing_FAQ
+Copyright: Copyright © 1995, 2022, Oracle and/or its affiliates.
+CopyrightUrl: https://www.oracle.com/legal/terms.html
+ShortDescription: VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use.
+Description: |-
+  VirtualBox is a powerful x86 and AMD64/Intel64 virtualization product for enterprise as well as home use.
+  Not only is VirtualBox an extremely feature rich, high performance product for enterprise customers, it is also the only professional solution that is freely available as Open Source Software under the terms of the GNU General Public License (GPL) version 2.
```
Comment (stephengillie) [inline manifests/o/Oracle/VirtualBox/7.0.4/Oracle.VirtualBox.locale.en-US.yaml:21]:
```suggestion
  Not only is VirtualBox an extremely feature rich, high performance product for enterprise customers, it is also the only professional solution whose binaries are freely available as Open Source Software under the terms of the GPL version 3.
```

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
With a new version of Winget in my testing VM, this package installs fully unattended. The new version generally installs dependencies before running the package installer. It should pass validation once the new version is in the pipelines. 
* https://github.com/microsoft/winget-cli/releases/tag/v1.4.3132-preview

**Pull request merged**